### DISCUSSION TITLE
syside would work with monaco editor?

### DISCUSSION THREAD

**Discussion by PatrickF-mbse** [sensmetry/sysml-2ls#10] [General]
Hi,    I was wondering if it could be somehow possible to use the check rule of syside inside the standalone monaco editor?  I heard is it the same used in vscode.
I am trying to create a webclient with Monaco Editor used to write sysml v2 code.  Curious to see if could be possible to add the syside checks?  
Thank you

**Comment by daumantas-kavolis-sensmetry** [comment]
It should certainly be possible. Similarly to [syside-vscode](https://github.com/sensmetry/sysml-2ls/tree/main/packages/syside-vscode), Monaco editor language client needs to be implemented.
